.github/workflows/format.yml

```
@@ -0,0 +1,49 @@
+name: Format
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  prettier:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.9.1
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
```

No need to put this in a matrix, as it's only running on one version
```suggestion
      - name: Use Node.js
        uses: actions/setup-node@v2
        with:
          node-version: 14
```

---

i actually had with out a matrix, but i have no idea why i switched it use one haha

---

Haha those things happen 😛